homeassistant/components/esphome/climate.py

```diff
@@ -234,6 +240,16 @@ def current_temperature(self) -> float | None:
         """Return the current temperature."""
         return self._state.current_temperature
 
+    @property
+    @esphome_state_property
+    def current_humidity(self) -> int | None:
+        """Return the current humidity."""
+        return (
+            self._state.current_humidity
+            if self._static_info.supports_current_humidity
+            else None
+        )
```

Comment:
For this code base we want ternaries that span multiple lines to be converted to if blocks